doc/modules/ROOT/pages/index.adoc

```diff
@@ -1,46 +1,119 @@
+Hazelcast configuration can be stored in the Config Map. You can install it with the following command:
 
-////
-Write the steps in your guide here by making each step a different section
-////
+[source, shell]
+----
+$ kubectl apply -f config.yaml
+----
 
-== Summary
+=== Creating Secret with Enterprise Key (Enterprise Only)
+Hazelcast Enterprise requires the Hazelcast Enterprise License Key. You can store it as Kubernetes Secret.
+
+[NOTE]
+====
+This step is not required for Hazelcast Open Source.
+====
+
+[source, shell]
+----
+$ kubectl create secret generic hz-enterprise-license --from-literal=key=<LICENSE-KEY>
+----
+
+=== Starting Hazelcast Cluster
+
+Finally, deploy the Hazelcast cluster:
+
+[source, shell]
+----
+$ kubectl apply -f hazelcast.yaml
+----
+
+For Hazelcast Enterprise, use the following command:
+
+[source, shell]
+----
+$ kubectl apply -f hazelcast-enterprise.yaml
+----
 
-// Provide a quick summary
+=== Persistent Volume
 
-== See Also
+This is a *prerequisite* step if you have JARs.
 
-// Add some links to resources, such as other related guides.
-// Use relative links used on the home page (see https://raw.githubusercontent.com/hazelcast-guides/guides-site/master/home/modules/ROOT/pages/index.adoc)
+In order to share a custom domain JARs (for example `EntryProcessor` implementations) among Hazelcast pods, you need to add a persistent volume in Kubernetes.
+
+There are many ways you can define and map volumes in Kubernetes. Types of volumes are discussed in the https://kubernetes.io/docs/concepts/storage/volumes/[official documentation].
+
+Once you have created a volume, copy your custom Hazelcast JARs into the root volume directory and add it into the `CLASSPATH` environment variable.
+
+In the following example a GCE Persistent Disk named "my-hz-disk" has been already created and populated with the custom configuration.
+
+Open a text editor and add the following deployment YAML for persistent volume:
+
+[source, yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: hz-pv
+spec:
+  capacity:
+    storage: 10Gi
+  storageClassName: standard
+  accessModes:
+    - ReadWriteOnce
+    - ReadOnlyMany
+  persistentVolumeReclaimPolicy: Retain
+  gcePersistentDisk:
+    pdName: my-hz-disk
+    fsType: ext4
+----
+
+Save this file as `hz-pv.yaml`. Please also notice that `Reclaim Policy` is set as `Retain`. Therefore, contents of this folder will remain as is, between successive `claims`.
+
+Create the persistent volume:
+
+[source, shell]
+----
+$ kubectl apply -f hz-pv.yaml
+----
+
+Please note that contents of your previous deployment is preserved. If you change the claim policy to `RECYCLE`, you have to transfer all custom files to `<your-pv-path>` before each successive deployments.
+
+Now edit `hazelcast.yaml`, and a PersistentVolumeClaim definition to match the above PersistentVolume. Finally, set the `CLASSPATH` env variable to a valid path and add a corresponding `volumeMount` in the `StatefulSet`.
```

Comment:
Isn't another `kubectl apply -f hazelcast.yaml` needed to apply changes? Or have you just ignored it in this section of the guide?

Reply:
Yeap, it's needed. Updated the guide